src/app/links/links.component.html

```diff
@@ -112,17 +111,18 @@ <h1 class="text-primary">{{ 'links.done' | translate }}</h1>
               ngxClipboard
               type="button"
             >
-              {{ 'links.copy' | translate }}
+              {{ 'links.admin.copy' | translate }}
             </button>
-            <button
-              (click)="navigate(absoluteAdminLink)"
-              class="btn btn-transparent text-primary btn-with-image"
+            <a
+              [routerLink]="adminLink"
+              rel="noopener noreferrer"
+              target="_blank"
+              class="btn btn-transparent text-primary link-with-image"
               data-id="adminLinkNavigate"
-              type="button"
             >
-              {{ 'links.open' | translate }}
-              <img alt="create" src="assets/icn_umfrage_erstellen_primary.svg">
-            </button>
+              {{ 'links.admin.open' | translate }}
+              <img alt="" src="assets/icn_umfrage_erstellen_primary.svg">
+            </a>
```

Comment:
Ab ~990px Bildschirmbreite bricht der Text in 2 Zeilen und das img ist dann nicht gecentered

![image](https://github.com/user-attachments/assets/da2ed573-8211-4b61-984b-bbecd5a6982a)